plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -124,6 +124,19 @@ func init() {
 		},
 	})
 	addControllerRole(rbac.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "expand-controller"},
```

Comment:
gate this on `utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes)`

Reply:
Feature gating the controller policy - somehow disables controller access to Update etc, even though feature gate is enabled for every component including apiserver.  I have enabled it via `export FEATURE_GATES=blah` and then `./hack/local-up-cluster.sh` and verified that apiserver cli does contain feature gate.

Reply:
As discussed offline, there appears to be a bug in the way policies are bootstapped. @liggitt cooked up a patch for me - https://gist.github.com/liggitt/8cbf0c907d06f4ca3c093073600f1ef7 which works.  But I am uncomfortable carrying a patch that I didn't write and understand. 

I think, we should leave this policy change  as it is, until the behaviour is fixed in master and then I am happy to feature gate this policy change.

Reply:
>  I am uncomfortable carrying a patch that I didn't write and understand.

https://github.com/kubernetes/kubernetes/pull/51818 is open to build the roles on demand, which would make feature gates effective during startup reconcile.

> I think, we should leave this policy change as it is, until the behaviour is fixed in master and then I am happy to feature gate this policy change.

Merging this ungated automatically initializes these roles/bindings on start, which is not what we want.